src/main/java/task/Task.java

```diff
@@ -0,0 +1,72 @@
+package task;
+
+/**
+ * Task class which can be extended to create sub tasks.
+ */
+public class Task {
+
+    /** Type of Task. */
+    protected String type;
+    /** Description of Task. */
+    protected String description;
+    /** Boolean indicated whether Task is done or not. */
+    protected boolean isDone;
+
+    /**
+     * Initialises a Task object.
+     * @param description Description of the Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Gets the status icon of the task. Shows a tick if done and an "x" if Task is not done.
+     * @return Status icon of the task.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "✓" : "✘");
+    }
+
+    /**
+     * Sets the Task as done or not depending on the argument.
+     * @param isDone boolean to determine whether Task is marked as done or not.
+     */
+    public void setDone(boolean isDone) {
```

Comment:
Maybe the method setDone can be used without a variable? i.e. setDone()
The name of method is self-explanatory as setting the task as Done.
If the variable boolean isDone is assigned incorrectly might have errors!